include/engine/box/box.h

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file box.h
+ * @author Grégory Brivady
+ * @brief Box template class definition
+ * @version 0.1.0
+ * @date 2022-08-25
+ * 
+ */
+#ifndef ENGINE_BOX_BOX_H
+#define ENGINE_BOX_BOX_H
+
+#include <cmath>
+#include <vector>
+// #include "engine/entity.h"
+typedef std::pair<double, double> point;
+
+// class Entity;
```

Comment:
is this normal?

Reply:
Absolutly not!
Fixed in edd0351f9e680154778cf996f59235c8d7c92c8a